CHANGELOG.md

```diff
@@ -1,3 +1,53 @@
+# 1.2.0
+- **refactor:** remove `dart_code_metrics` dependency
+- **refactor:** remove [banned-usage](https://dcm.dev/docs/rules/common/banned-usage/) `RichText` identifier in favour of [prefer-text-rich](https://dcm.dev/docs/rules/flutter/prefer-text-rich/)
+- **refactor:** increase `max-identifier-length` of [prefer-correct-identifier-length](https://dcm.dev/docs/rules/common/prefer-correct-identifier-length/) to 40
+
+- **fix:** set `ignore-blocs` to `true` for `dispose-fields` (BlocProvider automatically handles disposal of the Bloc)
```

Comment:
Important note: only those blocs, that were created via `create` method from `BlocProvider`, not via `BlocProvider.value`.